dev/io.openliberty.microprofile.openapi.2.0.internal/src/io/openliberty/microprofile/openapi20/internal/ApplicationProcessor.java

```diff
@@ -48,6 +48,7 @@
 import io.openliberty.microprofile.openapi20.internal.cache.ConfigSerializer;
 import io.openliberty.microprofile.openapi20.internal.services.ConfigFieldProvider;
 import io.openliberty.microprofile.openapi20.internal.services.ModelGenerator;
+import io.openliberty.microprofile.openapi20.internal.services.ModuleSelectionConfig;
```

Comment:
spurious import

Reply:
Previously ApplicationProcessor referenced ModuleSelectionConfig by being in the same package as it. But ModuleSelectionConfig became ModuleSelectionConfigImpl (its now declarative service) with an interface in a different package. So the import is needed.